src/debug_renderdoc.cpp

```diff
@@ -13,46 +13,14 @@
 
 namespace bgfx
 {
-	bool findModule(const char* _name)
+	void* findModule(const char* _name)
 	{
 #if BX_PLATFORM_WINDOWS
-		HANDLE process = GetCurrentProcess();
-		DWORD size;
-		BOOL result = EnumProcessModules(process
-						, NULL
-						, 0
-						, &size
-						);
-		if (0 != result)
-		{
-			HMODULE* modules = (HMODULE*)alloca(size);
-			result = EnumProcessModules(process
-				, modules
-				, size
-				, &size
-				);
-
-			if (0 != result)
-			{
-				char moduleName[MAX_PATH];
-				for (uint32_t ii = 0, num = uint32_t(size/sizeof(HMODULE) ); ii < num; ++ii)
-				{
-					result = GetModuleBaseNameA(process
-								, modules[ii]
-								, moduleName
-								, BX_COUNTOF(moduleName)
-								);
-					if (0 != result
-					&&  0 == bx::strCmpI(_name, moduleName) )
-					{
-						return true;
-					}
-				}
-			}
-		}
-#endif // BX_PLATFORM_WINDOWS
+		return (void*)GetModuleHandleA(_name);
```

Comment:
This code was added:
https://github.com/bkaradzic/bgfx/commit/d705fbc9bde4c7f23843a12ec8d079c0d2cc2e41

There was reason (I can't recall anymore) why I used this method instead simply calling `GetModuleHandleA`.

Reply:
OK, I will put that back.